packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -211,22 +263,43 @@ export class ClusterReachability {
   }
 
   /**
-   * Stores the latency in the result for the given protocol and marks it as reachable
+   * Saves the latency in the result for the given protocol and marks it as reachable,
+   * emits the "resultReady" event if this is the first result for that protocol,
+   * emits the "clientMediaIpsUpdated" event if we already had a result and only found
+   * a new client IP
    *
    * @param {string} protocol
    * @param {number} latency
+   * @param {string|null} [publicIp]
    * @returns {void}
    */
-  private storeLatencyResult(protocol: 'udp' | 'tcp' | 'xtls', latency: number) {
+  private saveResult(protocol: 'udp' | 'tcp' | 'xtls', latency: number, publicIp?: string | null) {
     const result = this.result[protocol];
 
     if (result.latencyInMilliseconds === undefined) {
       LoggerProxy.logger.log(
         // @ts-ignore
-        `Reachability:index#storeLatencyResult --> Successfully reached ${this.name} over ${protocol}: ${latency}ms`
+        `Reachability:index#saveResult --> Successfully reached ${this.name} over ${protocol}: ${latency}ms`
       );
       result.latencyInMilliseconds = latency;
       result.result = 'reachable';
+      if (publicIp) {
+        result.clientMediaIPs = [publicIp];
```

Comment:
Do we need to append the `publicIp` to `clientMediaIPs` here or is this intentional?

Reply:
it's intentional, because this is the first time we're saving the result (see the check in line 279) so there is no array to append to